CHANGELOG.md

```diff
@@ -21,6 +21,8 @@ Due to [\#5553](https://github.com/decidim/decidim/pull/5553), SSL is turned on
 
 ### Added
 
+- **decidim-participatory_processes**: Add related assemblies to participatory processes [\#5868](https://github.com/decidim/decidim/pull/5868)
+- **decidim-core**: Allow users to register with a preferred language. [\#5789](https://github.com/decidim/decidim/pull/5789
```

Comment:
Hi @armandfardeau 
Hadn't seen that that changelog line doesn't have to be there. Could you check please?
Aside, a parenthesis would be missing at the end of excess line.
After that, all will be ok!
Thanks for your job! 
Thanks!

Reply:
No problem, maybe a merge issue. Should be fixed now